tools/FxCopRules/DataWebRules.csproj

```diff
@@ -40,11 +69,16 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="FxCopSdk">
+      <HintPath>C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\PrivateAssemblies\FxCopSdk.dll</HintPath>

```

Comment:
I think this change may break for devs using VS2015. We should revert the file if it is not involved with the code change.

Reply:
Reverted it.